matrix/matrix.py

```diff
@@ -41,12 +49,12 @@ def _validate(self, data: tuple):
     @classmethod
     def unity(cls, dimension: int):
         """Generate a matrix with a main diagonal of 1's, elsewhere 0's."""
-        unity_matrix = []
+        unity_matrix = ()
 
         for i in range(dimension):
-            unity_matrix.append((0,) * i + (1,) + (0,) * (dimension - i - 1))
+            unity_matrix += ((0,) * i + (1,) + (0,) * (dimension - i - 1),)
```

Comment:
why not with list comprehension?

Reply:
You told me to seperate it. [https://github.com/osherdp/python-training/pull/60#discussion_r399812941](url)